apk-generator/v2/config.py

```diff
@@ -22,7 +22,14 @@ class Config:
     CELERY_BROKER_URL = os.environ.get('REDIS_URL', 'redis://localhost:6379/0')
     CELERY_RESULT_BACKEND = CELERY_BROKER_URL
     SECRET_KEY = os.environ.get('SECRET_KEY', 'JQgeEYk9b4VjxhJP')
-
+    EMAIL_SERVICE = os.environ.get('EMAIL_SERVICE', 'sendgrid')
```

Comment:
In one of the earlier reviews, you suggested to set it to 'send grid'. I'll make it none then and add a condition to check that as well?